articles/tech-resources/custom-bounce-domain.md

```diff
@@ -3,16 +3,22 @@ title: "Creating a Custom Bounce Domain"
 description: "You can improve your deliverability and email branding by adding a custom bounce domain A custom bounce domain provides another layer to the branding that some IS Ps care about Note Only one custom bounce domain is permitted per account at this time By adding a bounce domain to your..."
```

Comment:
Remove the note about only one bounce domain being available per account. That limitation is gone.